apps/ledger-live-desktop/src/renderer/families/evm/EditTransaction/Body.tsx

```diff
@@ -288,7 +289,7 @@ const Body = ({
     editType,
     transaction,
     transactionToUpdate,
-    status,
+    status: status as TransactionStatus,
```

Comment:
`as` don't seem necessary ?

Reply:
It is because the `TransactionStatus` from the props is generic while this is the `TransactionStatus` of coin-evm only. 
I tried to change it at the bridge level to have better typing with generics & without casting, but it's, again, a PR of hundreds of files... So it's for another time :p